src/dailies.ts

```diff
@@ -336,9 +336,7 @@ function prepFamiliars(): void {
         continue;
       }
       withProperty("autoBuyPriceLimit", priceCap, () => retrieveItem(1, drink));
-      if (have(drink)) {
-        visitUrl(`inventory.php?action=robooze&which=1&whichitem=${toInt(drink)}`);
-      }
+      if (have(drink)) Robortender.feed(drink);
```

Comment:
i am trying to track back through the code and follow the processing but i am unsure how, exactly, we attempt to feed the robort a drink again after we reminisce. this feels like a "maybe we solve it for your next run of garbo with a reminisce but not for the current run" solution. and it doesn't even -really- do that, because if the price of fish heads is above market cap, it won't ever retrieve item and convert them to drive-bys. i think you can solve this by just coercing garbo to always convert fish heads to drive-bys; it's slightly annoying behavior for some people i guess but i think it would at least solve this issue

Reply:
okay talked to you over DM, now i get the first part. one small thing we could do here is to call prep familiars twice -- once as part of dailies and once as your post-free-fight setup, in case you now have a fish head and can actually give it to the robort. it's not perfect but it's better than a "fix for tomorrow" solution i think.